addons/account/controllers/download_docs.py

```diff
@@ -60,3 +61,15 @@ def download_invoice_documents_filetype(self, invoices, filetype, allow_fallback
             zip_content = _build_zip_from_data(docs_data)
             headers = _get_headers(_('invoices') + '.zip', 'zip', zip_content)
             return request.make_response(zip_content, headers)
+
+    @http.route('/account/get_file_from_url', type='json', auth='user')
+    def get_file_from_url(self, url):
+        response = requests.get(url, timeout=10)
+        content_disposition = response.headers.get('Content-Disposition', '').split('; ')
+        if response.status_code == 200 and content_disposition[0] == 'attachment':
+            file_name = len(content_disposition) > 1 and content_disposition[1].replace('filename=', '')
+            return {
+                'content': response.content,
+                'content_type': response.headers.get('Content-Type'),
+                'file_name': file_name or "document",
+            }
```

Comment:
Done!
It seemed important at first because on portal of purchase order were saying customer to copy/paste url to create so
![image](https://github.com/user-attachments/assets/09443b3e-7cea-4b23-aedf-aa0b6173c2d0)
But I get you point we need to check security concerns and also we can add this in stable 18.0 after freeze

Keeping copy/pasting files as it look useful so user can copy/paste files from file manager to here and it does not have any issue related to security and file types and also user can select bunch of files from there and paste it in list or kanban view
Still we can drop the remaining part from copy/paste commit as currently it isn't easy to focus on list/kanban view full of records as there is not space to focus except footer.

what do you think?

Reply:
Yeah, ofc, copy/paste and drag & drop are useful and must be kept.
As long as we make sure that the user has the rights to create the targeted record (and we make sure we do not bypass the existing rights)